src/components/senior/DisplaySenior.tsx

```diff
@@ -48,6 +48,12 @@ const DisplaySenior = (props: DisplayProps) => {
       seniorId: senior.id,
     });
   };
+  const seniorFiles = senior.Files.map((file) => {
+    const userTimeZoneOffset = new Date().getTimezoneOffset();
+    const newDate = new Date(file.date.getTime() + userTimeZoneOffset * 60000);
+    const { date, ...other } = file;
+    return { date: newDate, ...other };
```

Comment:
nit: I think it's cleaner to do

```ts
return { ...file, date: newDate }
```

instead of de-structuring `{ date, ...other }`